Scripts/load_env_vars.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export CONTROLS_REPO=$(dirname $(dirname $(realpath "$BASH_SOURCE")))
```

Comment:
This seems finicky and will break if the script ever gets moved into a subdirectory. Maybe change it to `export CONTROLS_REPO=$(git -C $BASH_SOURCE rev-parse --show-toplevel)` which will work as long as the script is in the repo